packages/grafana-toolkit/src/cli/index.ts

```diff
@@ -138,9 +138,10 @@ export const run = (includeInternalScripts = false) => {
 
   program
     .command('plugin:build')
+    .option('--maxJestWorkers <num>|<string>', 'Limit number of Jest workers spawned')
     .description('Prepares plugin dist package')
     .action(async cmd => {
-      await execTask(pluginBuildTask)({ coverage: false, silent: true });
+      await execTask(pluginBuildTask)({ coverage: false, silent: true, maxJestWorkers: cmd.maxJestWorkers });
```

Comment:
Out of curiosity what happens if `cmd.maxJestWorkers` or `cmd.maxWorkers` is undefined in all these cases, do we need a default or is this just going to work?

Reply:
It works like before then, with Jest defaulting to ncpu - 1

Reply:
Ok thank you